docs/tutorials/plots.rst

```diff
@@ -1057,3 +1059,39 @@ make multiple plots:
     plt.show()
 
 :ref:`Return to Top <plots>`
+
+.. _finder_image:
+
+Finder Chart/Image
+==================
+
+`astroplan` includes a function for generating quick finder images from the
+command line, `~astroplan.plots.plot_finder_image`, by querying for images from
```

Comment:
With "from the command line" you mean the interactive Python prompt?
When I read "from the command line" I assume the terminal / shell, not Python, but I think that's not possible, right?
So can you rephrase here to make this more clear?